database/migrations/2022_01_05_151733_add_year_to_global_clusters.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddYearToGlobalClusters extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('clusters', function (Blueprint $table) {
+            $table->year('year')->nullable(true)->index();
```

Comment:
I don't think we need to call nullable = true? I think we can just use `nullable()`